### DISCUSSION TITLE
Fails in os build 22621.2283

### DISCUSSION THREAD

**Discussion by SpitsNApaches** [valinet/ExplorerPatcher#2054] [General]
Loving Explorer Patcher, thanks for your efforts. Will Dontate!

BUT, I just updated to Windows 11 build 22621.2283 and it made explorer patcher unstable, and a pop up showed saying explorer had crashed multipe times. 

Hope you manage to fix it as I have had my taskbar on the top of the screen since 2008! Cant live like this! 

Cheers!

**Comment by roadturtle** [comment]
Same happened to me. ExplorerPatcher is unusable right now.

**Comment by SpitsNApaches** [comment]
yeah. Donated 10 euros as I like the work! hopefully will be fixed.

**Comment by michele654** [comment]
Ditto...

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/2051
